src/django/api/views/reference_image.py

```diff
@@ -0,0 +1,46 @@
+from django.shortcuts import get_object_or_404
+
+from rest_framework import generics
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.permissions import IsAuthenticated
+
+from ..models import Boundary, ReferenceImage
+from ..models.user import Roles
+from ..serializers.reference_image import ReferenceImageSerializer
+
+
+class ReferenceImageList(generics.CreateAPIView):
+    permission_classes = [IsAuthenticated]
+    serializer_class = ReferenceImageSerializer
+
+    def get_queryset(self):
+        if self.request.user.role == Roles.VALIDATOR:
+            raise PermissionDenied("Validators cannot view image details.")
```

Comment:
Validators should be allowed to view images, as they may need to during inspection of the submission. They should not be allowed to edit or update them.

Reply:
[507d9ba](https://github.com/azavea/iow-boundary-tool/pull/114/commits/507d9ba645ec12e67dca470dd66f5a55d4335d89)